usr/lib/pkcs11/common/utility.c

```diff
@@ -282,8 +282,8 @@ CK_RV _UnlockMutex(MUTEX *mutex)
 
 CK_RV CreateXProcLock(char *tokname, STDLL_TokData_t *tokdata)
 {
-    CK_BYTE lockfile[2 * PATH_MAX + sizeof(LOCKDIR_PATH) + 6];
-    CK_BYTE lockdir[PATH_MAX + sizeof(LOCKDIR_PATH)];
+    char lockfile[2 * PATH_MAX + sizeof(LOCKDIR_PATH) + 6];
```

Comment:
2 * PATH_MAX + sizeof ... + 6 ???
what about simple use PATH_MAX and change the code some lines below to use snprintf instead of sprintf